jasminewd2/index.d.ts

```diff
@@ -44,6 +44,13 @@ declare namespace jasmine {
     toThrowError(expected?: new (...args: any[]) => Error | Promise<new (...args: any[]) => Error>, message?: string | RegExp | Promise<string | RegExp>): Promise<void>;
   }
 
+  interface ArrayLikeMatchers<T extends ArrayLike<T>> extends Matchers<T> {
```

Comment:
Okay, now I got it. 🛁 

Sorry I had a huge knot in my thinking and could not see past it 😅 

Of course you are correct, I will fix this right away